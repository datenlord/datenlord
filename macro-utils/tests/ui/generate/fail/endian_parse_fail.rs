use macro_utils::*;

fn main() {
    let _ = generate_bytes! {
        NE;
        u64: 65,
        u64: 20,
        u64: 507,
        u32: 0x20,
        u32: 0x20,
        str: b"foo.bar\0",
    };
}