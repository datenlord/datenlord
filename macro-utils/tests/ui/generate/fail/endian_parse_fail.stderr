error: expected `BE` or `LE`
 --> tests/ui/generate/fail/endian_parse_fail.rs:5:9
  |
5 |         NE;
  |         ^^
